ihmc-perception/src/main/java/us/ihmc/perception/imageMessage/CompressionType.java

```diff
@@ -0,0 +1,23 @@
+package us.ihmc.perception.imageMessage;
+
+import perception_msgs.msg.dds.ImageMessage;
+
+public enum CompressionType
+{
+   JPEG,
+   NVJPEG, // NVJPEG compression and OpenCV JPEG compression are not compatible. WHY NVIDIA, WHY?????
+   PNG,
+   NVCOMP,
+   ZSTD_NVJPEG_HYBRID,
+   UNCOMPRESSED;
+
+   public void packImageMessage(ImageMessage imageMessageToPack)
```

Comment:
In other places in the codebase, I called these type methods toMessage and fromMessage. I don't mind if you want to keep these names though.